packages/back-end/src/models/FeatureModel.ts

```diff
@@ -687,6 +687,23 @@ export async function editFeatureRule(
   });
 }
 
+export async function editFeatureRuleSet(
+  revision: FeatureRevisionInterface,
+  environment: string,
+  rules: FeatureRule[],
+  user: EventAuditUser
+) {
```

Comment:
nit: I'd suggest using an object payload here for readability and flexibility w.r.t. argument orders and such,